executor/analyze.go

```diff
@@ -735,6 +735,9 @@ func (e *AnalyzeFastExec) buildSampTask() (needRebuild bool, err error) {
 
 	store, _ := e.ctx.GetStore().(tikv.Storage)
 	e.cache = store.GetRegionCache()
+	if e.cache == nil {
+		return false, errors.Trace(errors.New("region cache is invalid, fast analyze abort"))
+	}
```

Comment:
Shall we check the result of `e.cache.GetRPCContext(bo, loc.Region)` in `getSampRegionsRowCount` as well?